package.json

```diff
@@ -28,6 +28,7 @@
     "@actions/io": "^1.1.3",
     "fp-ts": "^2.14.0",
     "node-fetch": "^3.3.1",
+    "proxy-agent": "^6.2.0",
```

Comment:
We still need to add `proxy-agent` to `tsup.config.js/noExternal` s.t. it gets bundled in `dist`.

Reply:
Done. Decent PR diff now 😂